app/views/posts/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<%if @user != nil%>
+
+	<% provide(:title, "#{@post.title}") %>
+
+	<h1><%= @post.title %><br></h1>
+	<p><%= simple_format(@post.body) %></p>
+	<br>
+	<br>
+	<p>
+	Post created by:<br>
```

Comment:
You did a good job. Can we do this?

```ruby
<% if logged_in? %>
                Post created by:<br>
		<%= @user.name %>
	<% end %>
```
I was thinking about what will happen if a user creates an account with the same name `ANNONOYMOUS`?